src/utils/EthDataProvider/EthDataProvider.js

```diff
@@ -157,6 +157,45 @@ const lockTokensTx = async (amount, time, wallet) => {
   return false;
 };
 
+const withdraw = async wallet => {
+  if (wallet.status === "connected") {
+    const provider = new ethers.providers.Web3Provider(wallet.ethereum);
+    const hiIQ = new ethers.Contract(
+      hiIQAddress,
+      hiIQAbi,
+      provider.getSigner()
+    );
+
+    const result = await hiIQ.withdraw({
+      gasLimit: 1000000
```

Comment:
is that gaslimit right? otherwise we can try without hardcoding gasLimit or setting the proper value

Reply:
I set 75000 as a gas limit